### PR TITLE
Google Sheets

### PR DESCRIPTION
**Pull request by michelle0927**

### CODE REVIEW

File: components/google_sheets/google_drive.app.js [hunk 1]
```diff
@@ -0,0 +1,235 @@
+const axios = require("axios");
```
Comment (jsendo) [inline components/google_sheets/google_drive.app.js:1]: this looks like a copy paste of https://github.com/PipedreamHQ/pipedream/blob/master/components/google_drive/google_drive.app.js
and it's out of date now.  Remove this file and just require the one in the google_drive directory.

File: components/google_sheets/sources/new-row-added/new-row-added.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
+const google_sheets = require("../../google_sheets.app.js");
+const google_drive = require("../../google_drive.app.js");
```
Comment (jsendo) [inline components/google_sheets/sources/new-row-added/new-row-added.js:2]: `const google_drive = require("../../../google_drive/google_drive.app.js");` should work here

File: components/google_sheets/sources/new-updates/new-updates.js [hunk 3]
```diff
@@ -0,0 +1,232 @@
+const google_sheets = require("../../google_sheets.app.js");
+const google_drive = require("../../google_drive.app.js");
```
Comment (jsendo) [inline components/google_sheets/sources/new-updates/new-updates.js:2]: same as the file above

File: components/google_sheets/sources/new-updates/new-updates.js [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+      return;
+    }
+
+    const { files, newPageToken } = await this.google_drive.getModifiedSheets(
+      pageToken,
+      this.drive === "myDrive" ? null : this.drive,
+      sheetIDs
+    );
+    for (const file of files) {
+      const spreadsheet = await this.google_sheets.getSpreadsheet(file.id);
```
Comment (jsendo) [inline components/google_sheets/sources/new-updates/new-updates.js:156]: maybe refactor this into functions?

**Review by jsendo — CHANGES_REQUESTED** [review]

File: components/google_sheets/google_drive.app.js (hunk 1 above)
Comment (michelle0927) [reply components/google_sheets/google_drive.app.js:1]: Much of it is copied from the one in the google_drive directory, but the last 3 methods are not. Maybe I could require the one in the google_drive directory in this app file, remove the copied methods, and reference the original methods (from the google_drive directory app file) in this app file?

**Pull request merged**